### DISCUSSION TITLE
Errors in Log chia

### DISCUSSION THREAD

**Discussion by Sitnnov** [Chia-Network/chia-blockchain#5321] [Support]
hello , my errors .

seldom------full_node chia.full_node.full_node: ERROR    Consensus error 124, not disconnecting
Often----------status 3 error: Err.DOUBLE_SPEND
seldom -------status 3 error: Err.COIN_AMOUNT_NEGATIVE ---always for one transaction

ERROR    Exception:  <class 'concurrent.futures._base.CancelledError'>, closing connection {'host': '113.236.16.35', 'port': 8444}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 435, in wait_for
concurrent.futures._base.CancelledError

Read the topics on this topic, did not understand anything, I am not an experienced user

- OS: WIN 10
- CPU : amd 3500
- Chia 1.1.5
help me please
![Файл 00001](https://user-images.githubusercontent.com/84305774/118450870-5260f880-b6fd-11eb-8090-37edd5b955b8.jpg)

**Comment by shaunlourdes84** [comment]
hello , my errors .

2021-05-17T14:11:07.183 full_node chia.full_node.full_node: ERROR    Consensus error 124, not disconnecting
2021-05-17T14:11:34.986 full_node chia.full_node.full_node: ERROR    Consensus error 124, not disconnecting
2021-05-17T14:11:58.192 full_node chia.full_node.full_node: ERROR    Consensus error 124, not disconnecting

Read the topics on this topic, did not understand anything, I am not an experienced user help me please

OS: WIN 10
CPU : amd 3700x
Chia :1.1.5

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
